app/src/main/java/com/example/cmput301f18t26/icare/Controllers/DataController.java

```diff
@@ -1,12 +1,15 @@
 package com.example.cmput301f18t26.icare.Controllers;
 
 class DataController {
-    private static final DataController ourInstance = new DataController();
 
-    static DataController getInstance() {
-        return ourInstance;
+    public DataController() {
     }
 
-    private DataController() {
+    public void fetch(){
+
+    }
+
+    public void save(){
+        
```

Comment:
You will need to mock out getters and setters for the private DS's that you choose to use.

Reply:
Also, I think that that save will need to return the String: id that is assigned by ElasticSearch.